packages/electron/package.json

```diff
@@ -20,12 +20,8 @@
     "build:typescript": "tsc",
     "license-checker": "npx license-checker --production --exclude MIT,Apache-2.0,Unlicense,BSD-2-Clause,BSD-3-Clause",
     "generate-attribution": "npx generate-attribution",
-    "build:react": "npx run-script-os",
-    "build:react:win32": "build-scripts\\install-react-deps.bat",
-    "build:react:default": "./build-scripts/install-react-deps.sh",
-    "build:csharp": "npx run-script-os",
-    "build:csharp:win32": "build-scripts\\install-csharp.bat",
```

Comment:
What's the difference between " win32",  "default", and "build:csharp"?

Reply:
@xueningl-az, is from the run-script-os command, more details here but this doesn't run properly on newer versions of npm/node. https://github.com/charlesguse/run-script-os

Reply:
"build:csharp", "build:react" will build the csharp and react packages respectively. "win32"/"default" tells which script to run, .bat for windows and .sh for linux